addons/web/static/src/core/notebook/notebook.scss

```diff
@@ -1,18 +1,81 @@
 .o_notebook {
-    &.horizontal {
-        flex-direction: column;
+    --notebook-link-active-border-color: #{$border-color};
+    --notebook-link-hover-border-color: #{$gray-200};
+
+    .o_notebook_headers {
+        margin: 0 var(--notebook-margin-x, 0);
+    }
+
+    .nav {
+        padding: 0 var(--notebook-padding-x, 0);
+        background-color: $o-white;
+
+    }
 
-        .nav.nav-tabs {
-            flex-flow: row nowrap;
+    .nav-item {
+        white-space: nowrap;
+
+    }
+    
+    .nav-link {
+        background-color: $o-view-background-color;
+        border: 1px solid var(--notebook-link-border-color, transparent);
```

Comment:
Something is off here, not active tabs should have a border-bottom (not sure if this is the affected line).
<img width="499" alt="image" src="https://user-images.githubusercontent.com/8558351/186757272-10da5dfa-7e54-4efc-9d20-a713ede95e46.png">